### ISSUE TITLE
TWRP 3.3.1-4 for guacamole does not mount Data partition after Android 10 update

### ISSUE THREAD

**Issue by Dave1482** [TeamWin/Team-Win-Recovery-Project#1512]
- [✓] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [✓] I am running the latest version of TWRP
- [✓] I have read the FAQ (https://twrp.me/FAQ/)
- [✓] I have searched for my issue and it does not already exist

**Device codename**: guacamole
**TWRP version**: 3.3.1-4

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?
Update to OxygenOS 10.0.0
Boot into recovery
Try mounting Data

#### WHAT IS THE EXPECTED RESULT?
Mounted Data partition successfully

#### WHAT HAPPENS INSTEAD?
Unable to mount Data partition

#### ADDITIONAL INFORMATION
Works fine for Pie, broken on Android 10

**Comment by Aaditya-oneplus7pro** [comment]
Hi, i am haing the same problem that after flashing the android 10 rom from twrp 3..3.1-4 and flashing the twrp and magisk zip file my twrp is not showing internal storage. Magisk manager is not showing magisk is installed. What should i do, please help

**Comment by strugee** [comment]
Duplicate of #1503?

**Comment by Dave1482** [comment]
No, as I am talking about the OnePlus 7 Pro, codenamed," guacamole"

**Comment by Aaditya-oneplus7pro** [comment]
Hi Dave 1482, I found the solution for guacamole (codename of OnePlus 7 pro). You need to install the TWRP 3.3.1-65 with .65 IMG and then install the android 10 rom. After that you will not get the internal storage 0 error.

**Comment by Dave1482** [comment]
I've figured that out before, but it isn't readily available on
https://twrp.me/oneplus/oneplus7pro.html
I also forgot I made this post lol

**Comment by Aaditya-oneplus7pro** [comment]
If your device doesn't starts then use the msm v4 tool with L2 Certification from latest XDA forum and reset your device to original factory settings and then root the device with TWRP 65. The after install the latest Android 10 rom after that you will not get this internal storage 0 error

**Comment by doctor-whooves** [comment]
I have the same issue. Do I flashboot flash boot TWRP 3.3.1-65? How do I install the android 10 rom if I can't get to internal storage?

**Comment by Dave1482** [comment]
you boot to twrp and swap to other slot. I've made a guide to root/install TWRP on the OnePlus 7 Pro on the OnePlus forum
